examples/inlineHello_cluster.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: activemonitor.keikoproj.io/v1alpha1
+kind: HealthCheck
+metadata:
+  generateName: inline-hello-
+  namespace: test
```

let's stick with the `health` namespace for cluster level for now... but really, we should probably make namespace at healthcheck and workflow levels optional and default to `health` when not provided
```suggestion
  namespace: health
```